modules/ROOT/pages/dynamic-evaluate-component-reference.adoc

```diff
@@ -21,7 +21,158 @@ The script can use any of the usual xref:dataweave-variables-context.adoc[contex
+
+To accomplish this example you must create two DataWeave scripts and store them locally, create the Mule application, run and test the application.
+
+The following screenshot shows the Anypoint Studio app flow for this example:
+
+.Dynamic Evaluate Flow example in Anypoint Studio
+image::dynamic-evaluate-flow-1.png[Dynamic Evaluate Flow example in Anypoint Studio]
+
+=== Create and Store the DataWeave Scripts
+
+To create and store the DataWeave scripts locally, follow the next steps:
+
+. Create a new Mule project in Studio.
+. In the *Package Explorer* window, under your Mule project right-click the *src/main/resources* folder and select *New>File*.
+. In the *File name* field, add the name `dw_en-US.dwl` to create the new DataWeave script file.
+. Click *Finish*.
+. In the *dw_en-US.dwl* window, paste the following DataWeave expression:
++
+[source,dataweave,linenums]
+----
+%dw 2.0
+output application/json
+---
+{
+    "First Name": payload.name,
+    "Last Name": payload.last,
+    "Country": payload.loc
+}
+----
++
+. Repeat the previous steps and create a new file named `dw_es-AR.dwl` that contains the following DataWeave expression:
++
+[source,dataweave,linenums]
+----
+%dw 2.0
+output application/json
+---
+{
+    "Nombre": payload.name,
+    "Apellido": payload.last,
+    "País": payload.loc
+}
+----
++
+. Save your changes.
+
+=== Create the Mule Application
+
+To create the Mule application in Studio, follow the next steps:
+
+. In the *Package Explorer* window, under the *src/main/mule (Flows)* folder, navigate to your flow `.xml` file.
+. In the *Mule Palette* view, select the *HTTP Listener* source and drag it onto the canvas.
+The source initiates the flow by listening for incoming HTTP requests.
+. Set the *Path* field to `/test`.
+. Click the plus sign (*+*) next to the *Connector configuration* field to configure a global element that can be used by all instances of the HTTP Listener in the app.
+. On the *General* tab, configure the following fields:
++
+* *Host*: `All Interfaces [0.0.0.0] (default)`
+* *Port*: `8081`
++
+. Click *OK*.
+. Drag the *File Read* operation to the right of *HTTP Listener*.
+. Click the plus sign (*+*) next to the *Connector configuration* field to configure a global element that can be used by all instances of the File Read operation in the app.
+. On the *General* tab, set the *Working Directory* field to the main resources folder where your Studio project and previous DataWeave scripts live, for example:
++
+`/Users/username/AnypointStudio/studio-workspace/dynamicevaluate/src/main/resources`
++
+. Click *OK*.
+. In the *File path* field set the following path `#["dw_" ++ attributes.queryParams.locale ++ ".dwl"]`. +
+The connector reads the locally stored DataWeave script file names that start with `dw_` and contains the `locale` query parameter (`attributes.queryParams.locale`) received in the HTTP request.
++
+. In the *Advanced* tab, set the *Target Variable* field with `dwScript`. +
```

Comment:
```suggestion
. In the *Advanced* tab, set the *Target Variable* field to `dwScript`. +
+
```

Reply:
Don't forget to insert a linespace between a step (or bullet-list item) and its further description. GLOBAL